tiledb/sm/config/config.cc

```diff
@@ -80,6 +79,9 @@ const std::string Config::CONFIG_LOGGING_LEVEL = "1";
 const std::string Config::CONFIG_LOGGING_LEVEL = "0";
 #endif
 const std::string Config::CONFIG_LOGGING_DEFAULT_FORMAT = "DEFAULT";
+const std::string Config::PROFILE_HOMEDIR =
+    tiledb::common::filesystem::home_directory();
```

Comment:
Hmmm. This makes me a little uneasy since it is a "volatile" default, i.e. different users will see different things. This might lead to trouble if a user wants to export their configs for another user to use, for example.

Perhaps we can make this `std::optional<std::string>` so that we have the same fixed "None" value to mean "use the home directory?

Is it fair to say that the value of these configurations will be the values passed to the internal RestProfile constructor?

Reply:
Answered in `config.cc:966`.
Yes - the values of these parameters are used to load a profile when trying to look up a config parameter.

Reply:
Default for homedir and name were changed to empty string following an internal discussion. I guess that this solves your concern here?

Reply:
Not relevant anymore.